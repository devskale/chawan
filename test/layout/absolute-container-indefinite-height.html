<!DOCTYPE html>
<pre style="position: relative">
<div style="position: absolute; bottom: 0">bottom .1</div>
outer 1
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
<div style="position: absolute; bottom: 0">bottom 2</div>
</pre>
<pre style="position: relative">
<div style="position: absolute; bottom: 0">bottom .3</div>
outer 2
<div style="position: absolute; bottom: 0">bottom 4</div>
</pre>
