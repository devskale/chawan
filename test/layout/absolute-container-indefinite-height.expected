

outer 1
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
.
bottom 21


outer 2
bottom 43
